build_dep.sh

```diff
@@ -5,22 +5,16 @@
 # This source code is licensed under Apache 2.0 License,
 # attached with Common Clause Condition 1.0, found in the LICENSES directory.
 #
-# step 1: ./build_dep.sh <C|U|N>
+# step 1: ./build_dep.sh [N] # N: no need packages
```

Comment:
I think if the user provides no parameter, it's better to be the same as --help. let the use confirm the operations.

how about ./build_dep.sh -y & ,.build_dep.sh -n

Reply:
The script header already explains the usage, most customers do not use the parameters